developing/how-to-update-openstack-upgrade-path.md

```diff
@@ -46,6 +46,30 @@ the debian package `bind9`, and we can use `rmadison bind9` to find all the
 versions for charm `designate-bind`. Similarly, for charm `gnocchi`, the
 versions can be found using `cmadison gnocchi`.
 
+The list of package need to be updated are:
+
+```shell
+# Install rmadison
+apt install devscripts
+# Install cmadison
+snap install cmadison
+
+# Find the upper version for the specific releases
+rmadison gnocchi
+rmadison bind9
+rmadison rabbitmq-server
+
+# The following are found in https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html
+# - mysql
+# - vault
+# - ovn
+
+# The following are found in https://docs.ceph.com/en/latest/releases/
+# - ceph
+
+# The rest of the openstack components are found in https://releases.openstack.org/index.html
+```
```

Comment:
These could be outside the codeblock. :)

Reply:
yep